iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/user.component.html

```diff
@@ -66,7 +66,7 @@ <h1>
 
                         <!-- Send reset password email or change password-->
                         <user-password user="$ctrl.user"></user-password>
-
+                        <aup-resign user="$ctrl.user"></aup-resign>
```

Comment:
This object is visible to both admins and users but admins can enter user's page and will find this button that acts for theirselves and not for the user. This object should probably be hidden in case of `!$ctrl.isMe() && $ctrl.isVoAdmin()`. Could you check this behavior and confirm this?

Reply:
Yes, I have checked if admin enter user's page admin can see the button.